common/config.go

```diff
@@ -78,5 +79,10 @@ func GetConfig() Config {
 		config.LogLevel = &level
 	}
 
+	if config.LogOutput == "" {
+		output := "StdOutAndFile"
+		config.LogOutput = output
```

Comment:
Is there any reason why you can't do
```go
config.LogOutput = "StdOutAndFile"
```

Reply:
Looking at the code above this I guess you were just copying the format, it isn't needed for this because you don't need to take the pointer of the string.

Reply:
Addressed in commit [4bbb966](https://github.com/WiiLink24/wfc-server/pull/48/commits/4bbb96663419071f4abc63ffaa3526081035c403).